internal/build/docker.go

```diff
@@ -0,0 +1,104 @@
+package build
+
+import (
+	"archive/tar"
+	"bytes"
+	"context"
+	"io"
+	"regexp"
+	"strings"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+)
+
+type Mount struct {
+	Repo          LocalGithubRepo
+	ContainerPath string
```

Comment:
nit: i always read this as "path TO container" rather than "path IN container", and so I used `RemotePath` in my code to disambiguate. If you want 🤷‍♀️

Reply:
Yeah, except in this case it's not remote at all. And since we're getting passed the Dockerfile we don't need to worry about the path to the dockerfile! :D